test/simple_malloc-ut.sh

```diff
@@ -132,6 +132,12 @@ check "./ddprof --initial-loaded-libs-check-delay 500 ./test/simple_malloc --use
 # Test slow profiler startup
 check "env DD_PROFILING_NATIVE_STARTUP_WAIT_MS=200 ./ddprof ./test/simple_malloc ${opts}" 1
 
+# Test follow execs
+check "./ddprof bash -c \"./test/simple_malloc ${opts}\"" 1
+
+# Test disabling follow execs
+check "env DD_PROFILING_NATIVE_ALLOCATION_PROFILING_FOLLOW_EXECS=0 DD_PROFILING_NATIVE_STARTUP_WAIT_MS=200 ./ddprof bash -c \"./test/simple_malloc ${opts}\"" 0 0 "inuse-space"
```

Comment:
Interesting to test the removal of the feature. For overhead, there might be a use case where users do not want to follow some execs. Good thinking.